<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImportStepSimplification</class>
 <widget class="QDialog" name="DialogImportStepSimplification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Step Simplification</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Classification method</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="volumeRadioButton">
        <property name="text">
         <string>Volume</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="similarityRadioButton">
        <property name="text">
         <string>Similarity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="complexityRadioButton">
        <property name="text">
         <string>Complexity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Degree of simplification</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="1">
       <widget class="QDoubleEdit" name="keepPercentLineEdit">
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="keepPercent">
        <property name="text">
         <string>Percentage of parts kept</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="keepPercentHorizontalSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="filePathGroup">
     <property name="title">
      <string>File paths</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2">
       <widget class="QToolButton" name="inputToolButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../openscantools.qrc">
          <normaloff>:/icons/100x100/save.png</normaloff>:/icons/100x100/save.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRegExpEdit" name="outputLineEdit">
        <property name="placeholderText">
         <string>Simplified step file output folder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output folder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRegExpEdit" name="inputLineEdit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Step file path to simplify</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="outputToolButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../openscantools.qrc">
          <normaloff>:/icons/100x100/open_project.png</normaloff>:/icons/100x100/open_project.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input path</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="simplificateButton">
       <property name="text">
        <string>Simplificate and Import</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>115</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
  <customwidget>
   <class>QRegExpEdit</class>
   <extends>QLineEdit</extends>
   <header>regexpedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>inputLineEdit</tabstop>
  <tabstop>inputToolButton</tabstop>
  <tabstop>outputLineEdit</tabstop>
  <tabstop>outputToolButton</tabstop>
  <tabstop>volumeRadioButton</tabstop>
  <tabstop>similarityRadioButton</tabstop>
  <tabstop>complexityRadioButton</tabstop>
  <tabstop>keepPercentHorizontalSlider</tabstop>
  <tabstop>keepPercentLineEdit</tabstop>
  <tabstop>simplificateButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
